thrift/thrift-gen/generate.go

```diff
@@ -78,7 +79,13 @@ func runThrift(inFile string, outDir string) error {
 
 	// Generate the Apache Thrift generated code.
 	goArgs := fmt.Sprintf("go:thrift_import=%s,package_prefix=%s", *apacheThriftImport, *packagePrefix)
-	if err := execThrift("-r", "--gen", goArgs, "-out", outDir, inFile); err != nil {
+	args := []string{
+		"--gen", goArgs, "-out", outDir, inFile,
+	}
+	if *thriftNoRecurse == false {
```

Comment:
Style for bool comparisons is generally to obviate the constants like `!*thriftNoRecurse`.